src/plugins/data_source/server/plugin.ts

```diff
@@ -67,6 +72,19 @@ export class DataSourcePlugin implements Plugin<DataSourcePluginSetup, DataSourc
       return dataSourcePluginStart.getAuthenticationMethodRegistry();
     });
 
+    const { editMode } = config;
+
+    if (editMode && editMode !== EditMode.None) {
+      const dataSourcePermissionWrapper = new DataSourcePermissionClientWrapper(editMode);
+
+      // Add data source permission client wrapper factory
+      core.savedObjects.addClientWrapper(
+        2,
```

Comment:
```suggestion
        ORDER_FOR_DATA_SOURCE_EDIT_MODE_WRAPPER,
```
No magic number, better to assign to a meaningful variable, and I do not know if it will conflict with workspace wrapper, we can use a larger number to avoid that.

Reply:
Sure, I will update.